DOCUMENTATION.md

```diff
@@ -37,10 +37,10 @@ The module exports a single factory function.
 **Summary**: Creates a new SDK instance using the default or the provided options.  
 **Example**  
 ```js
-import { getSdk } from 'balena-sdk';
+const {getSdk} = require('balena-sdk');
```

Comment:
The sdk since v15 is now an es module and should be imported as one.
Maybe we can keep the current style as well as the old require style to make it easier for node consumers.
```suggestion
// with es6 imports
import { getSdk } from 'balena-sdk';
// or with node require
const { getSdk } = require('balena-sdk');
```

Reply:
Keeping backward compatibility is a great idea @thgreasi 
Also, have we documented the part regarding building the docs. Do you mind if we pen down a CONTRIBUTING of some sort for documentation changes? Thanks!